Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# run multiple instances of the 'px4' binary, but w/o starting the simulator.
+# It assumes px4 is already built, with 'make px4_sitl_default'
+
+# The simulator is expected to send to TCP port 4560+i for i in [0, N-1]
+# For example gazebo can be run like this:
+#./Tools/gazebo_sitl_multiple_run.sh -n 10 -m iris
+
+function cleanup() {
+	pkill -x px4
+	pkill gzclient
+}
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ]
+then
+	echo "Usage: $0 [-n <num_vehicles>] [-m <vehicle_model>]"
+	exit 1
+fi
+
+while getopts n:m: option
+do
+	case "${option}"
+	in
+		n) NUM_VEHICLES=${OPTARG};;
+		m) VEHICLE_MODEL=${OPTARG};;
+	esac
+done
+
+num_vehicles=${NUM_VEHICLES:=3}
+export PX4_SIM_MODEL=${VEHICLE_MODEL:=iris}
+
+if [ "$PX4_SIM_MODEL" != "iris" ]
+then
+	echo "Currently only iris vehicle model is supported!"
+	exit 1
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+src_path="$SCRIPT_DIR/.."
+
+build_path=${src_path}/build/px4_sitl_default
+mavlink_udp_port=14560
+mavlink_tcp_port=4560
+world="empty"
+
+echo "killing running instances"
+pkill -x px4 || true
+
+sleep 1
+
+source ${src_path}/Tools/setup_gazebo.bash ${src_path} ${src_path}/build/px4_sitl_default
+
+echo "Starting gazebo"
+gzserver ${src_path}/Tools/sitl_gazebo/worlds/${world}.world --verbose &
+sleep 5
+
+n=0
+while [ $n -lt $num_vehicles ]; do
```

Comment:
That's a detail, and I'm not an expert in bash, but couldn't we do a for loop here? So that it fits on one line and removes the `n=$(($n + 1))` at the end of the loop.

```suggestion
for n in {0..$((num_vehicles-1))}; do
```

Reply:
That seems better. but wouldn't there be value on being consistent with https://github.com/PX4/Firmware/blob/master/Tools/sitl_multiple_run.sh#L25 ?

Reply:
Right, your call :wink:.